meson.build

```diff
@@ -18,6 +18,21 @@ project(
   ],
 )
 
+# https://mesonbuild.com/Python-module.html
+py_mod = import('python')
+py3 = py_mod.find_installation(pure: false)
+py3_dep = py3.dependency()
+
+# Emit a warning for 32-bit Python installs on Windows; users are getting
+# unexpected from-source builds there because we no longer provide wheels.
+is_windows = host_machine.system() == 'windows'
+if is_windows and py3.has_variable('EXT_SUFFIX')
```

Comment:
Should we also raise the warning on 32-bit linux?

Reply:
I don't think so. People who use that on purpose usually know what they are doing.

Reply:
Indeed, because the usual source of a 32-bit python on 64-bit Linux is something like "I explicitly enabled multiarch support and then installed `python3:i386` instead of `python3`". It's actually extremely hard to do by accident.